community/examples/fsi-montecarlo-on-batch.yaml

```diff
@@ -0,0 +1,126 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: fsi-montecarlo-on-batch
+
+vars:
+  project_id: ## Set GCP Project ID Here ##
+  deployment_name: fsimontecarlo
+  region: us-central1
+  zone: us-central1-a
+
+deployment_groups:
+- group: setup
+  modules:
+  - id: enable_apis
+    source: community/modules/project/service-enablement
+    settings:
+      gcp_service_list: [
+        "bigquery.googleapis.com",
+        "cloudresourcemanager.googleapis.com",
+        "container.googleapis.com",
+        "logging.googleapis.com",
+        "notebooks.googleapis.com",
+        "batch.googleapis.com",
+        "pubsub.googleapis.com",
+        "compute.googleapis.com"
+      ]
+
+- group: primary
+  modules:
+
+  - id: new-notebook
+    source: community/modules/compute/notebook
+    use:
+      [fsi_bucket,bq-table, pubsub_topic]
+    settings:
+      create_tutorial: fsi
+      name_prefix: notebook
+      machine_type: n1-standard-4
+
+  - id: fsi_bucket
+    source: community/modules/file-system/cloud-storage-bucket
+    settings: 
+      name_prefix: fsi_bucket
+      random_suffix: true
+      force_destroy: true
+      local_mount: /home/jupyter/fsi
+
+  - id: fsi_tutorial_files
+    source: ./community/modules/files/fsi-montecarlo-on-batch
+    use: [bq-dataset, bq-table, fsi_bucket, pubsub_topic]
+    settings:
+
+  - id: pubsub_topic
+    source: community/modules/pubsub/topic
+    settings:
+      topic_id: null
```

Comment:
null is the same as just not defining. Why not just leave out of the blueprint. here and elsewhere.

Reply:
It's weird, I have a default value but it complains about a required value, if I don't have this as null. Is there something I am doing wrong?

Reply:
When do you see the error? I was able to run without it and got no errors. Maybe you need to call `make`.